src/chain.h

```diff
@@ -307,6 +308,16 @@ class CBlockIndex
     //! Will be std::nullopt if and only if nChainTx is zero.
     std::optional<CAmount> nChainOrchardValue;
 
+    //! Change in value held by the development fund lockbox over this block.
+    //!
+    //! Will be std::nullopt for older blocks until a reindex has taken place.
+    std::optional<CAmount> nLockboxValue;
```

Comment:
It should not be necessary for this to be optional. We know that the lockbox deltas and value balance are zero for every block before activation of the NU that introduces the lockbox, and an "older block" is by definition prior to that.

Reply:
In general, I tried to write this such that it would also be correct for the genesis block; I guess that I need to check for what the block subsidy of the genesis block is, and if it's hardcoded to zero then this can be set to zero as well.

Reply:
It is zero because of the slow start, and moreover it can't be spent (because of a pre-existing Bitcoin bug when we forked, which we didn't fix because we knew it would be zero).